tests/cli/cdk_smoke_tests/cdk-basic.bats

```diff
@@ -39,7 +39,7 @@ setup_file() {
 @test "Build and deploy connector" {
     # Build
     cd $CONNECTOR_DIR
-    run $CDK_BIN build -- --target x86_64-unknown-linux-gnu
+    run $CDK_BIN build
```

Comment:
Now this is provided here: https://github.com/infinyon/fluvio/pull/3222/files#diff-2b406cc64ffbfe71ad848e32377e6e988548a47845782d6df9ddc9ce0dffacf8R37